### PR TITLE
docs: minor change to standardising README wording

### PR DESCRIPTION
**Pull request by amfage**
#### Motivation

Just a small tidy up for clarity.

#### Modification

Change wording for providers fields.

#### Checklist

_If not applicable, provide explanation of why._

- [ ] Tests updated N/A
- [X] Docs updated
- [ ] Issue linked in Title N/A

### CODE REVIEW

File: workflows/raster/README.md [hunk 1]
```diff
@@ -30,10 +30,10 @@ In addition, a Basemaps link is produced enabling visual QA.
...
+| producer               | enum  | Unknown                               | Imagery producer :warning: Ignored if `producer_list` is used.                                                                                                                                                        |
+| producer_list          | str   |                                       | List of imagery producers, separated by semicolon (;). :warning: Has no effect unless a semicolon delimited list is entered.                                                                                          |
+| licensor               | enum  | Unknown                               | Imagery licensor. :warning: Ignored if `licensor_list` is used.                                                                                                                                                       |
+| licensor_list          | str   |                                       | List of imagery licensors, separated by semicolon (;). :warning: Has no effect unless a semicolon delimited list is entered.                                                                                          |
```
Comment (l0b0) [inline workflows/raster/README.md:36]: Wouldn't a missing semicolon result in an error rather than "no effect", since it's an enum?

Comment (paulfouquet) [reply workflows/raster/README.md:36]: It is a string that is parsed in `topo-imagery`

**Review by paulfouquet — APPROVED** [review]

File: workflows/raster/README.md (hunk 1 above)
Comment (amfage) [reply workflows/raster/README.md:36]: The `licensor_list` is not an enum.

Comment (l0b0) [reply workflows/raster/README.md:36]: > The `producer_list` and `licensor_list` are not enums.

OIC, ignore me.

**Review by l0b0 — APPROVED** [review]

**Pull request merged**